cookbooks/crystal_preferred_orientation_olivine_fraters_billen_2021/doc/crystal_preferred_orientation_olivine_fraters_billen_2021.md

```diff
@@ -0,0 +1,119 @@
+This cookbook describe how to use ASPECT CPO implementation to calculate the major
+olivine fabrics observed in the lab experiments. It focuses on reproducing Fig. 3
+of {cite}`fraters_billen_2021_cpo`.
+
+## Motivation
+The Earth's plates can move relative to each other because the underlying mantle
+can be deformed to accommodate such motions.
+This motion reflects dynamics inside the planet but cannot be observed directly, as the depth of interest is beyond human drilling capability. However, indirect observations from seismic anisotropy provide information about the directionality of seismic wave speeds propagating inside the Earth. If such directionality is controlled by the mantle’s motion or deformation, we can use seismic anisotropy observations to infer mantle deformation.
+
+It’s important to note that this cookbook does not directly compare model outputs with seismic anisotropy observations. Instead, it provides the full elastic tensor, including its symmetry axes and fast directions, at the locations of particles in the model. To make direct comparisons with seismic observations, you would still need to process these results through a separate code that simulates seismic wave propagation through the modeled medium.
+
+Olivine is a major mineral of the Earth's upper mantle where continuous deformation
+takes place. People care about olivine fabric developments under simple shear as it may
+provide a bridge that links seismic anisotropy observations to the
+flows in the upper mantle that accommodate plate motions. Indeed, high temperature and
+pressure lab experiments using [Griggs apparatus](https://en.wikipedia.org/wiki/Griggs_apparatus) to investigate olivine fabrics developments
+under simple shear do find systematic types of fabrics based on the stress and water content conditions
+(see {cite:t}`karato2008` for details). Note that the effects of secondary phases, such as enstatite  {cite:t}`kaminski2004` can also be included in the method, we here only focus on olivine for simplicity and the input file also include example of enstatite calculation setup.
+
+## Model setup
+
+Following {cite:t}`fraters_billen_2021_cpo`, we prescribe simple shear in a 3d Cartesian box/cube
+with dimensions of $1 \times 1 \times 1 $ $[m^3]$. The shear strain rate
+$\dot{\epsilon}_{xz} = -5\times 10^{-6} [s^{-1}]$. The olivine/enstatite particle is placed
+right at the center of the cubic box so it stays stationary. The DRex implementation
+keep tracks of rotations of crystal grains within the particle under macroscopic deformation.
+
+```{figure-md} fig:model_setup_3D_box
+<img src="model_setup_3D_box.png" style="width:80.0%" />
+
+3D shear box with velocity vectors. The grey ball is the central olivine particle.
+```
+
+The model shows how crystal grains rotate and align under simple shear with pole figures.
+
+## The input file
+
+Instead of solving Stokes equation with initial and boundary conditions, we instead prescribe
```

Comment:
The Stokes equation doesnt have an initial condition.
```suggestion
Instead of solving the Stokes equation with boundary conditions, we instead prescribe
```